lib/src/Uncertainty/Model/openturns/DistributionImplementation.hxx

```diff
@@ -886,13 +891,29 @@ protected:
 
 #ifndef SWIG
 
+public:
   // Class used to wrap the computePDF() method for interpolation purpose
   class PDFWrapper: public EvaluationImplementation
   {
+    CLASSNAME
   public:
+    PDFWrapper()
+      : EvaluationImplementation()
+      , p_shared_distribution_(NULL)
```

Comment:
cant we use the default ctor here ? else use nullptr because NULL is for C

Reply:
Ok let's try it

Reply:
done